todo.sh

```diff
@@ -1485,14 +1485,22 @@ note: PRIORITY must be anywhere from A to Z."
 "listaddons" )
     if [ -d "$TODO_ACTIONS_DIR" ]; then
         cd "$TODO_ACTIONS_DIR" || exit $?
+        VALID=0
         for action in *
         do
             if [ -f "$action" ] && [ -x "$action" ]; then
                 echo "$action"
+                VALID+=1
             elif [ -d "$action" ] && [ -x "$action/$action" ]; then
                 echo "$action"
+                VALID+=1
             fi
         done
+        if ! [ "$VALID" -gt 0 ]; then
+             echo "TODO: '$TODO_ACTIONS_DIR' does not contain valid actions."
+        fi 
+    else
+        [ "$TODOTXT_VERBOSE" -gt 0 ] && echo "TODO: '$TODO_ACTIONS_DIR' does not exist." 
```

Comment:
I had to test for verbosity here because otherwise there would occur issues with the completion, which would add the error message to the list of available completions:  

```* FAIL 1: all arguments
expected 53 completion(s), got 59:
--- expect	2023-01-21 11:06:09.853173003 +0000
+++ compreply	2023-01-21 11:06:09.853173003 +0000
@@ -34,6 +34,12 @@
 replace
 report
 shorthelp
+TODO:
+/home/user/repos/todo.txt-cli/tests/trash
+directory.t6000-completion/.todo/actions
+does
+not
+exist.
 -@
 -@@
 -+
*   ok 2: arguments beginning with a
*   ok 3: all options
*   ok 4: all actions after command action
```
I would appreciate some help with the completion.